modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/DLCContentPage.java

```diff
@@ -54,6 +55,36 @@ public String deleteFunction() throws IOException {
     }
 
     /**
+     * Delete all the messages in DeadLetter Channel
+     */
+    public void deleteAllDLCMessages() {
+
+        String deletingMessageID = null;
+
+        try {
+            deletingMessageID = driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                     .getElement("mb.dlc.first.message.id"))).getText();
+            if(StringUtils.isNotBlank(deletingMessageID)) {
+                log.info("delete all dlc messages");
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.table.choose.all.box.xpath")))
+                                                    .click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.table.delete.button"))).click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.function.confirm"))).click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.function.success"))).click();
+            }
+
+        }catch (Exception e) {
+            log.error("unable to find any valid messages in dead letter channel.");
```

Comment:
I believe what hemika pointing out is not about try catch general logic, but rather about checking this condition of table being empty in a different way so that this exception will never be thrown here. Also if we're anyway printing this log message it should be info or warn not an error since we're just responding to a delete command and we find no elements to delete.